onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -234,13 +234,29 @@ void QnnLogging(const char* format,
                 QnnLog_Level_t level,
                 uint64_t timestamp,
                 va_list argument_parameter) {
-  ORT_UNUSED_PARAMETER(level);
   ORT_UNUSED_PARAMETER(timestamp);
 
-  // Always output Qnn log as Ort verbose log
   const auto& logger = ::onnxruntime::logging::LoggingManager::DefaultLogger();
-  const auto severity = ::onnxruntime::logging::Severity::kVERBOSE;
   const auto data_type = ::onnxruntime::logging::DataType::SYSTEM;
+
+  ::onnxruntime::logging::Severity severity;
+  switch (level) {
```

Comment:
>switch (level) [](http://example.com/codeflow?start=2&length=14)

The reason we always set QNN log as verbose log is for op validation case, QNN report error if the node failed QNN op validation API. QNN EP fallback that node to CPU. So, we shouldn't treat it as a Ort error.
QNN EP should control the log itself based on it's API calls, instead of transparently mapping QNN logs with it's log level.

Reply:
Do you think there is a way we can handle this special case? It sort of sucks that we lose QNN severity level since it's useful sometimes. If not, I can revert this aspect, but was hoping not

Reply:
So the safer thing is to revert this. However, for posterity IMO the log levels should be mostly mapped as-is from QNN -> ORT. The verbosity level is generally good information you don't want to lose. The CPU fallback comes as an error from QNN, and the argument presented here is that it shouldn't be an error, but perhaps informational. Reverting this part for now

Reply:
Discussed with Hector. Reverting this specific change for now due to the above referenced issue

Reply:
Also, reverting this means that if someone increases log level to info, then that increases QNN log level to INFO. However, they will never see that log message because QNN logs come out as verbose, and thus they will never see the INFO logs. Bummer